TrainingApp/src/store/logStateMiddleware.js

```diff
@@ -0,0 +1,8 @@
+const LogStateMiddleware = store => next => action => {
+    console.log('dispatching', action);
+    let result = next(action);
+    console.log('next state', store.getState().auth);
```

Comment:
this needs to removed

Reply:
I am using this for my development purpose to see the states and actions for easy debugging